src/main.js

```diff
@@ -20,8 +21,14 @@ async function loadAragonClient (network = 'main') {
 }
 
 async function start (mainWindow) {
-  // Start IPFS first so we can pin it afterwards
-  await ipfsInstance.start()
+  try {
+    const version = await ipfsInstance.api.ipfsApi.version()
+    console.log('Detected running instance of IPFS, no need to start our own')
+  } catch (e) {
+    console.log('Could not detect running instance of IPFS, starting it ourselves...')
+    await ipfsInstance.start()
+    startedIpfs = true
+  }
```

Comment:
what about

```js
await ipfsInstance.api.ipfsApi.version()
  .catch(() => {
    startedIpfs = true
    return ipfsInstance.start()
  })
```

Reply:
Hmm, I think the `try / catch` is more obvious as an either/or situation.